beacon-chain/blockchain/service.go

```diff
@@ -226,6 +196,43 @@ func (c *ChainService) updateHead(slot uint64) {
 	c.candidateCrystallizedState = nilCrystallizedState
 }
 
+func (c *ChainService) getParentSlot(block *types.Block) (uint64, error) {
+	parentBlock, err := c.chain.getBlock(block.ParentHash())
+	if err != nil {
+		return 0, err
+	}
+
+	return parentBlock.SlotNumber(), nil
+}
+
+func (c *ChainService) doesParentExist(block *types.Block) bool {
+	parentExists, err := c.chain.hasBlock(block.ParentHash())
+	if err != nil {
+		log.Debugf("Could not check existence of parent hash: %v", err)
```

Comment:
Why is this `Debugf`? Wouldn't `Errorf` be more appropriate here?

Reply:
Yeah I agree. I think it was always this way, but I'll switch these to `Errorf`